api/test/wfl/integration/datarepo_test.clj

```diff
@@ -77,20 +78,86 @@
             (is (= 1 row_count))
             (is (= 0 bad_row_count))))))))
 
-(def ^:private testing-dataset "3b41c460-d994-47a4-9bf7-c59861e858a6")
+(def ^:private testing-dataset "ff6e2b40-6497-4340-8947-2f52a658f561")
 
 ;; Get row-ids from BigQuery and use them to create a snapshot.
 (deftest test-create-snapshot
   (let [tdr-profile (env/getenv "WFL_TDR_DEFAULT_PROFILE")
-        {:keys [dataProject] :as dataset} (datarepo/dataset testing-dataset)
-        table     "flowcell"
-        start-datetime "2021-03-17"
-        end-datetime   "2021-03-19"
-        row-ids (->> (datarepo/make-snapshot-query dataset table start-datetime end-datetime)
-                     (bigquery/query-sync dataProject)
-                     :rows flatten)]
-    (testing "creating snapshot"
-      (fixtures/with-temporary-snapshot
-        (snapshots/unique-snapshot-request tdr-profile dataset table row-ids)
-        #(let [snapshot (datarepo/snapshot %)]
-           (is (= % (:id snapshot))))))))
+        dataset     (datarepo/dataset testing-dataset)
+        table       "flowcell"
+        from        "2021-03-30"
+        until       "2021-03-31"
+        row-ids     (-> (datarepo/query-table-between
+                         dataset
+                         table
+                         [from until]
+                         [:datarepo_row_id])
+                        :rows
+                        flatten)]
+    (fixtures/with-temporary-snapshot
+      (snapshots/unique-snapshot-request tdr-profile dataset table row-ids)
+      #(let [snapshot (datarepo/snapshot %)]
+         (is (= % (:id snapshot)))))))
```

Comment:
You could bind like they are used: `[from until] ["2021-03-30" "2021-03-31"]`
Or just inline them.